src/discord-cluster-manager/cogs/leaderboard_cog.py

```diff
@@ -251,76 +273,132 @@ async def leaderboard_create(
             try:
                 date_value = datetime.strptime(deadline, "%Y-%m-%d %H:%M")
             except ValueError:
-                # If that fails, try parsing just the date (will set time to 00:00)
-                date_value = datetime.strptime(deadline, "%Y-%m-%d")
+                try:
+                    date_value = datetime.strptime(deadline, "%Y-%m-%d")
+                except ValueError:
+                    raise ValueError(
+                        "Invalid date format. Please use YYYY-MM-DD or YYYY-MM-DD HH:MM"
+                    )
 
+            # Ask the user to select GPUs
+            view = GPUSelectionView([gpu.name for gpu in GitHubGPU])
+
+            await interaction.response.send_message(
```

Comment:
Shouldn't this be `interaction.followup.send(...)`. If there are going to be more messages in this interaction, then `interaction.response.send_message(...)` will cause problems.

Reply:
I've wrapped this around our followup_wrapper util to avoid this issue.